src/C/gdal.jl

```diff
@@ -345,53 +433,23 @@ Open a raster file as a GDALDataset.
 end
 
-
-"""
-    GDALOpenEx(const char * pszFilename,
-               unsigned int nOpenFlags,
-               const char *const * papszAllowedDrivers,
-               const char *const * papszOpenOptions,
-               const char *const * papszSiblingFiles) -> friend GDALDatasetH
-
-Open a raster or vector file as a GDALDataset.
-
-### Parameters
-* **pszFilename**: the name of the file to access. In the case of exotic drivers this may not refer to a physical file, but instead contain information for the driver on how to access a dataset. It should be in UTF-8 encoding.
-* **nOpenFlags**: a combination of GDAL_OF_ flags that may be combined through logical or operator. 
-
-Driver kind: GDAL_OF_RASTER for raster drivers, GDAL_OF_VECTOR for vector drivers. If none of the value is specified, both kinds are implied. 
-
-Access mode: GDAL_OF_READONLY (exclusive)or GDAL_OF_UPDATE. 
-
-Shared mode: GDAL_OF_SHARED. If set, it allows the sharing of GDALDataset handles for a dataset with other callers that have set GDAL_OF_SHARED. In particular, GDALOpenEx() will first consult its list of currently open and shared GDALDataset's, and if the GetDescription() name for one exactly matches the pszFilename passed to GDALOpenEx() it will be referenced and returned, if GDALOpenEx() is called from the same thread. 
-
-Verbose error: GDAL_OF_VERBOSE_ERROR. If set, a failed attempt to open the file will lead to an error message to be reported.
-* **papszAllowedDrivers**: NULL to consider all candidate drivers, or a NULL terminated list of strings with the driver short names that must be considered.
-* **papszOpenOptions**: NULL, or a NULL terminated list of strings with open options passed to candidate drivers. An option exists for all drivers, OVERVIEW_LEVEL=level, to select a particular overview level of a dataset. The level index starts at 0. The level number can be suffixed by "only" to specify that only this overview level must be visible, and not sub-levels. Open options are validated by default, and a warning is emitted in case the option is not recognized. In some scenarios, it might be not desirable (e.g. when not knowing which driver will open the file), so the special open option VALIDATE_OPEN_OPTIONS can be set to NO to avoid such warnings. Alternatively, since GDAL 2.1, an option name can be preceded by the @ character to indicate that it may not cause a warning if the driver doesn't declare this option.
-* **papszSiblingFiles**: NULL, or a NULL terminated list of strings that are filenames that are auxiliary to the main filename. If NULL is passed, a probing of the file system will be done.
-
-### Returns
-A GDALDatasetH handle or NULL on failure. For C++ applications this handle can be cast to a GDALDataset *.
-"""
-function GDALOpenEx(pszFilename, nOpenFlags::UInt32, papszAllowedDrivers, papszOpenOptions, papszSiblingFiles)
+function GDALOpenEx(pszFilename, nOpenFlags, papszAllowedDrivers, papszOpenOptions, papszSiblingFiles)
```

Comment:
did we lose the docstring here?

Reply:
Good catch, fixed in 5e8633e7a4ab6d834bb4c8cd356427a6f54c2113, I picked the wrong XML node with name GDALOpenEx here.